app/graphql/types/delete_item_type.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Types
+  class DeleteItemType < Base::Object
+    I18N_PATH = 'graphql.types.delete_item_type'
+
+    graphql_name 'DeleteItemType'
+    description I18n.t("#{I18N_PATH}.desc")
+
+    field :deleted_item_id, String, null: false, description: I18n.t("#{I18N_PATH}.fields.deleted_item_id")
```

Comment:
Nit: better to use `ID` type instead of `String`. `ID` works well with both incremental ids and UUIDs